datasources/src/test/java/org/opensearch/sql/datasources/transport/TransportUpdateDataSourceActionTest.java

```diff
@@ -59,8 +60,18 @@ public void testDoExecute() {
         .onResponse(updateDataSourceActionResponseArgumentCaptor.capture());
     UpdateDataSourceActionResponse updateDataSourceActionResponse =
         updateDataSourceActionResponseArgumentCaptor.getValue();
+
+    JsonResponseFormatter<CreateUpdateDatasourceResponse> jsonResponseFormatter =
+        new JsonResponseFormatter<>(JsonResponseFormatter.Style.PRETTY) {
+          @Override
+          protected Object buildJsonObject(CreateUpdateDatasourceResponse response) {
+            return response;
+          }
+        };
     Assertions.assertEquals(
-        "Updated DataSource with name test_datasource", updateDataSourceActionResponse.getResult());
+        jsonResponseFormatter.format(
```

Comment:
assert value should be literal string. it is much readable.

Reply:
Fixed in: [767a9cb](https://github.com/opensearch-project/sql/pull/2156/commits/767a9cbd4e2465506f1a9e007d632c6ca06662c5)